src/components/popovers/createEditTabpopovers/drawSelectedButton/properties/EditTypeDrawing.jsx

```diff
@@ -77,6 +78,10 @@ export default function EditTypeDrawing() {
     dispatch(changetoggleEditTypeDrawer(false));
   };
 
+  const changeSearchPropertyInputValue = (val) => {
+    setSearchPropertyTypeInput(val);
+  };
```

Comment:
why you need this changeSearchPropertyInputValue function if it only do setSearchPropertyTypeInput(val);